simulator-starter/src/test/java/com/consol/citrus/simulator/dictionary/InboundXmlDataDictionaryTest.java

```diff
@@ -26,15 +27,16 @@
             "</v1:TestRequest>");
 
     @Test
+    @Ignore // PFTODO fix and remove ignore
```

Comment:
gottcha! 😉

Reply:
Missed that one. Thx.  [fixed](https://github.com/citrusframework/citrus-simulator/pull/102/commits/db575b643e80593b32ce4c6f3c7f524933bba383)